source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,167 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+.. versionadded:: 2.3.0
+
+The Kontrol S3 is an introductory 4 deck controller with good build
+quality and integrated sound card. This is the first controller released
+with the "S3" name.
+
+The Kontrol S3 can run from USB bus power. Using the separate power
+supply increases the brightness of the LEDs, which is helpful for using
+it in daylight, and increases the volume of the headphone output.
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The Kontrol S3 is a USB class compliant audio and HID device,
+so it is compatible with Mixxx without any proprietary drivers on
+GNU/Linux and Mac OS X. On Windows, it is recommended to install the
+`driver from Native
+Instruments <https://www.native-instruments.com/en/support/downloads/drivers-other-files/>`__
+and select the ASIO sound API in the Sound Hardware section of Mixxx's
+Preferences.
+
+With the S3 plugged in, a MIDI device is listed as an available
+controller in Mixxx's Preferences. The controller uses
+HID for the knobs, buttons, and other components on the device, so the
+mapping can only be loaded when you select the HID device on the left
+side of Mixxx's Preferences.
+
+Mapping description
+-------------------
+
+Note that Mixxx doesn't have the concept of a single "master" deck for
+sync. Instead, push and hold the sync button to "lock" sync on for all
+decks you want to remain in sync. Or you can push Shift + Sync to lock
+sync on. Refer to `the Mixxx
+manual <http://www.mixxx.org/manual/2.0/chapters/djing_with_mixxx.html#master-sync>`__
```

Comment:
This is an absolute URL of a Mixxx 2.0 manual page. This link should be relative.

Reply:
done